### PR TITLE
Localize launch file changes for indoors and outdoors scenarios

### PR DESCRIPTION
**Pull request by Fedezac**
Changes made to `localize.launch` and `rs_odometry.launch` to manage both indoors and outdoors localization stacks, in a single launch file.
New `initial_state.yaml` parameter file, so that the initial position can be explicited only once for both EKFs in the dual EKF setup (outdoors scenario).
Added `gps` frame's static transform.
Removed `actuation_to_twist` node from `wheel_odometry.launch`.

### CODE REVIEW

File: src/svea_sensors/launch/localize.launch [hunk 1]
```diff
@@ -33,19 +35,32 @@
     <!-- Start wheel encoders -->
     <group if="$(arg use_wheel_encoders)">
         <include file="$(find svea_sensors)/launch/wheel_odometry.launch">
-            <arg name="start_serial" value="false"/>
+            <arg name="start_serial" value="true"/>
```
Comment (kaarmu) [inline src/svea_sensors/launch/localize.launch:38]: We usually start serial in top-level so this should be false. Maybe good to remove serial all together from `wheel_odometry.launch`.

File: src/svea_sensors/params/robot_localization/initial_state.yaml
Comment (kaarmu) [inline src/svea_sensors/params/robot_localization/initial_state.yaml:1]: Hmm, how does this differ from `initial_pose_x`, `..._y`, etc? Why should we have it in a separate rosparam file? isn't it better to have it accessible in your top-level launch file?

If we should have it like this, comment what the elements represent (X, Y, Z, ...)

**Review by kaarmu — CHANGES_REQUESTED** [review]

**Comment by kaarmu** [comment]
Also, I think this is a good PR for @Annika-wyt to look through/contribute with.

File: src/svea_sensors/launch/wheel_odometry.launch [hunk 2]
```diff
@@ -13,12 +13,6 @@
     <arg name="encoder_topic" default="lli/encoder" />
     <arg name="direction_topic" default="actuation_twist" />
 
-
-    <!-- Velocity estimation -->
-    <group if="$(arg start_serial)">
```
[the remark this answers is not in the document]
Comment (Fedezac) [reply src/svea_sensors/launch/wheel_odometry.launch:18]: Yes, it is launched in `rs_odometry.launch`

File: src/svea_sensors/params/robot_localization/initial_state.yaml
Comment (Fedezac) [reply src/svea_sensors/params/robot_localization/initial_state.yaml:1]: We moved everything in the `localize.launch` using only `initial_pose_x`, initial_pose_y`, initial_pose_a`

**Review by kaarmu — APPROVED** [review]

**Pull request merged**